### DISCUSSION TITLE
Dream Router , KPN , IPTV freezes

### DISCUSSION THREAD

**Discussion by calciumdude** [fabianishere/udm-iptv#134] [Q&A] [answered]
I have replaced my EdgerouterX  with a Dream Router and followed this setup but my SetupBoxes are now freezing .
they worked for a good day or so..

am i doing something wrong ?

**Comment by fabianishere** [comment, marked as answer]
Please provide diagnostic info as indicated by the guide. I can't help you with this little information.

**Reply by calciumdude** [reply]
I have found the issue myself..
Multicast was switched off at the VLAn in Unifi Network.
Thank you for doing all this work !!
Appreciate it a lot
